features/polyExplorer/src/components/clusterStories/messengers/details.jsx

```diff
@@ -0,0 +1,683 @@
+import React from "react";
+// import i18n from "../../../i18n";
+import SectionTitle from "../sectionTitle.jsx";
+import { ClusterSections } from "../clusterSections";
+import { StoryParagraph } from "./storyParagraph";
+import LinesChart from "../../dataViz/linesChart.jsx";
+import { BUBBLES_SPEECH_SIZES } from "../../../constants";
+
+import "./details.css";
+
+const Details = ({ data }) => {
+    const activeUsers = {
```

Comment:
Would it be possible to turn this into an array/hash and a loop? There seems to be some code repeated here...